Neos.Flow/Classes/Persistence/PersistenceManagerInterface.php

```diff
@@ -159,7 +159,7 @@ public function remove($object): void;
      * @throws UnknownObjectException
      * @api
      */
-    public function update($object): void;
+    public function update(object $object): void;
 
     /**
      * Adds the given object to a list of allowed objects which may be persisted when persistAll() is called with the
```

Comment:
Note (to self): Adjust this line in case https://github.com/neos/flow-development-collection/pull/2585 gets merged